tests/data/dnos_z9100-on.json

```diff
@@ -15486,7 +15486,7 @@
                     "entPhysicalIsFRU": "true",
                     "entPhysicalModelName": "Z9100-ON-01-FE-34",
                     "entPhysicalVendorType": "zeroDotZero",
-                    "entPhysicalSerialNum": "89PDDZ2",
```

Comment:
Seems like a regression

Reply:
Yeah, I think I remember this one, it pulls the express service code from a different oid and shoves it in there... 
I think I punted that one because it seems so weird...